model.py

```diff
@@ -0,0 +1,101 @@
+# generated by datamodel-codegen:
+#   filename:  application_schema.yaml
+#   timestamp: 2024-03-12T19:55:46+00:00
+
+from __future__ import annotations
+
+from datetime import date
+from typing import List
+
+from pydantic import AnyUrl, AwareDatetime, BaseModel, ConfigDict, Field, RootModel
+
+
+class Link(BaseModel):
+    model_config = ConfigDict(
+        extra='forbid',
+    )
+    title: str
+    url: AnyUrl
+
+
+class ReductionOfTheAttackSurfaceComponent(BaseModel):
+    model_config = ConfigDict(
+        extra='forbid',
+    )
+    date: date
+
+
+class SourceControlProtectionComponent(BaseModel):
+    model_config = ConfigDict(
+        extra='forbid',
+    )
+    url: AnyUrl
+    date: AwareDatetime
+
+
+class Settings(BaseModel):
+    model_config = ConfigDict(
+        extra='forbid',
+    )
+    desired_level: str = Field(..., alias='desired level')
+    application: str
+    team: str
+
+
+class ConductionOfSimpleThreatModelingOnTechnicalLevelComponent(BaseModel):
```

Comment:
In addition to `ConductionOfSimpleThreatModelingOnTechnicalLevelComponent` DSOMM has `Conduction of simple threat modeling on business level` or `Conduction of advanced threat modeling`.

The level often depends on 
- quality
- frequency

I wouldn't focus in the start on quality and just on frequency. The frequency/level shouldn't influence the name which leads to a generic name like `conduction of threat modeling`.